src/SimpleMdeReact.tsx

```diff
@@ -1,3 +1,8 @@
+/*
+ * DCSO Sentral
+ * Copyright (c) 2021 DCSO GmbH
```

Comment:
I don't think adding this is necessary:
https://github.com/RIP21/react-simplemde-editor/blob/master/LICENSE

Reply:
You are correct, my git commit hooks still had automatic default copyrights set, sorry about that.
Additionally i reworked the changes to use `useState()` insteaf of `useRef()` hook. It worked both ways but it seems useEffect with refs in dependency list got marked as anti-pattern.

Reply:
If so, may need to rename `textRef` to `textEl` or something else? I confused because you set `initialOptions.element` without `.current` but has suffix `Ref`.